test/test-mda-models/test-machine-learning-model/aissemble-machine-learning-inference/pyproject.toml

```diff
@@ -39,6 +39,11 @@ grpcio-testing = "^1.50.0"
 kappa-maki = ">=1.0.1"
 pylint = "^3.1.0"
 
+[tool.pylint.'MESSAGES CONTROL']
+extension-pkg-whitelist = "pydantic"
+
+[tool.pylint.MASTER]
+ignore-patterns = '.*pb2[\S]*.py'
```

Comment:
I: This locates the files with pb2[_base].py files generated by protobuff and ignores them. Pylint errors out due to unable to locate the referenced classes within these files.

Reply:
Let's add this comment inline since it's not super clear. Something like `#ignores protobuf generated files that fail linting`

Reply:
Added, thanks Emily!